### DISCUSSION TITLE
Check Sum error in Nozbe Cask

### DISCUSSION THREAD

**Discussion by rdasgup** [Homebrew/discussions#2885] [Casks]
Hello Team,

when I try to install Nozbe via home brew, I run into a check sum error for the downloaded packet.

Who shal I turn to so, this can be corrected?

Sheers
Roni

**Comment by carlocab** [comment]
Please see: https://github.com/Homebrew/homebrew-cask#reporting-bugs

**Comment by neutric** [comment]
Looks like they updated the zip file as `Nozbe.app` is no longer at `Mac (Developer ID) 2021-08-19 10-25-56/Nozbe.app` but at the top level. That would explain the checksum change.

Run `brew fetch nozbe`, which should give you something like this:

```
==> Downloading https://files2.nozbe.com/3201/release/Nozbe.app.zip
######################################################################## 100.0%
Downloaded to: /Users/ah/Downloads/HOMEBREW_CACHE/downloads/40f7002fc6130b44f137b4b93baeca25b9ed91d58b9d4027c28a363c828b0520--Nozbe.app.zip
SHA256: 3e35a6cecff0ebfbce4e4f04c6955b4900de92b89eb514b34a012c6f77772b7c
Warning: Cask reports different sha256: 1cf4461e172b66b486c24e1dbc6177559e921bf10e100b74262a165e44cee876
```

The cask definition needs to be updated with the new `sha256` checksum as seen above and `app "Mac (Developer ID) 2021-08-19 10-25-56/Nozbe.app"` needs to be changed to `app "Nozbe.app"`. This addresses the current problem, but the Pull Request is going to get stuck on `livecheck` reporting a new version which I couldn't find a download link for. The app might have gone app store only.

**Reply by rdasgup** [reply]
Hello neutric,

many thanks for your investigation and explanation.
So I will exclude Nozbe from handling with home brew.

Sheers
Roni

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/118396
